tests/components/slide_local/conftest.py

```diff
@@ -33,25 +34,22 @@ def mock_config_entry() -> MockConfigEntry:
 
 
 @pytest.fixture
-def mock_slide_api():
+def mock_slide_api() -> Generator[AsyncMock]:
     """Build a fixture for the SlideLocalApi that connects successfully and returns one device."""
 
-    mock_slide_local_api = AsyncMock()
-    mock_slide_local_api.slide_info.return_value = SLIDE_INFO_DATA
-
     with (
         patch(
-            "homeassistant.components.slide_local.SlideLocalApi",
+            "homeassistant.components.slide_local.coordinator.SlideLocalApi",
             autospec=True,
-            return_value=mock_slide_local_api,
-        ),
+        ) as mock_slide_local_api,
         patch(
             "homeassistant.components.slide_local.config_flow.SlideLocalApi",
-            autospec=True,
-            return_value=mock_slide_local_api,
+            new=mock_slide_local_api,
```

Comment:
Could this patch `goslideapi.goslideapi.GoSlideLocal` instead of multiple places in the integration?

Reply:
In general we want integrations to patch the libraries at the places they are used. https://docs.python.org/3/library/unittest.mock.html#where-to-patch